src/diffusers/models/transformers/transformer_sd3.py

```diff
@@ -330,6 +331,89 @@ def _set_gradient_checkpointing(self, module, value=False):
         if hasattr(module, "gradient_checkpointing"):
             module.gradient_checkpointing = value
 
+    def _load_ip_adapter_weights(self, state_dict: Dict, low_cpu_mem_usage: bool) -> None:
```

Comment:
we should probably make a loaderMixin for transformers at some point but can be a future PR (if we need to implement more loading related methods)

Reply:
Oh I see, similarly to `UNet2DConditionLoadersMixin`? For now we just have this one method, but I can do it now, less things to move around in the future. Should it be in a new file like `diffusers/loaders/transformers.py`?

Reply:
maybe `diffusers/loaders/transformers_sd3.py` because I think flux adapter would needs its own method, no?

Reply:
Right, and it's also more consistent with the existing code structure as well, thanks for pointing it out :)

Reply:
Indeed Flux IP Adapter has its own method, we can also rename `IPAdapterJointAttnProcessor2_0` to `SD3IPAdapterJointAttnProcessor2_0`, I've used `FluxIPAdapterJointAttnProcessor2_0` for Flux.

Reply:
Just updated `IPAdapterJointAttnProcessor2_0` to `SD3IPAdapterJointAttnProcessor2_0`, and created `SD3Transformer2DLoadersMixin`.  Maybe docs could be improved a bit, but it only has `_load_ip_adapter_weights` so far :sweat_smile: